queries.sql

```diff
@@ -36,3 +36,41 @@ SELECT neutered,MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MAX(weight_kg), MIN(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-12-31' AND '2000-12-31' GROUP BY species;
 
+UPDATE animals SET species_id = 1 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 1 WHERE name NOT LIKE '%mon';
+
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon','Pikachu');
+UPDATE animals SET owner_id = 3 WHERE name IN ('Charmander','Squirtle','Blossom');
+UPDATE animals SET owner_id = 4 WHERE name IN ('Angemon','Boarmon');
+
+SELECT animals.name
+FROM animals
+JOIN owners ON owners.id = animals.owner_id 
+WHERE owners.full_name = 'Melody Pond';
+SELECT animals.name
+FROM animals
+JOIN species ON species.id = animals.species_id 
+WHERE species.name = 'Pokemon';
+SELECT owners.full_name ,animals.name
+FROM animals
+RIGHT JOIN owners ON owners.id = animals.owner_id;
+SELECT species.name, COUNT(*)
+FROM animals
+JOIN species ON animals.species_id = species.id
+GROUP BY (species.name);
+SELECT owners.full_name, animals.name, species.name
+FROM animals
+JOIN owners ON owners.id = animals.owner_id
+JOIN species ON animals.species_id = species.id
+WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
+SELECT owners.full_name, animals.name, animals.escape_attempts
+FROM animals
+JOIN owners ON owners.id = animals.owner_id
+WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
```

Comment:
- [ ] Kindly include the screenshot for this query as I have not seen in your pull request description. Or kindly label the screenshots properly as I have requested you to do in the pull request summary

![image](https://user-images.githubusercontent.com/86318284/213216405-e98c1939-8604-4bd8-9d72-ec8e93a810bb.png)